combat-trainer.lic

```diff
@@ -715,6 +715,8 @@ class SpellProcess
       end
     end
 
+    waitrt? while XMLData.prepared_spell == 'Vivisection' && game_state.use_stealth_attack? && ! hide?
```

Comment:
I think `DRSpells.prep_spell` is a better way to check the spell name, rather than using the XMLData itself.